oaknational/Oak-Web-Application
# fix: set school to not listed in onboarding [LESQ-1180]

## Description

Music year: 2021

- Set schoolName and schoolId to `notListed` in local storage when selecting no school during onboarding

## Issue(s)

Fixes LESQ-1180

## How to test

1. Go to https://deploy-preview-3044--oak-web-application.netlify.thenational.academy
2. Click on a lesson and click sign in to download button
3. Sign up and go through onboarding journey _as a completely clean user_, selecting no school
4. You should see `{"schoolName":"notListed","schoolId":"notListed"}` in cookies in browser
5. Repeat the process with and without opting into Hubspot mailing lists during onboarding.

## Screenshots

How it used to look (delete if n/a):
<img width="405" alt="image" src="https://github.com/user-attachments/assets/dc2d683b-cd40-4264-863f-c05f327641d3">

How it should now look:
<img width="456" alt="image" src="https://github.com/user-attachments/assets/38aba3a9-693a-43b4-b1dc-f1601e217162">

## Checklist
